_community_projects/gekatateam-neptunus.markdown

```diff
@@ -0,0 +1,9 @@
+---
+name: Neptunus
+description: Neptunus is written in Go data processing pipelines engine for consuming, transforming and producing events that can be used as a Logstash alternative.
+owner: Gekatateam
+link: https://github.com/gekatateam/neptunus
+license: Apache 2.0
+license_link: https://github.com/gekatateam/neptunus/blob/main/LICENCE
+owner_link: https://github.com/organizations/gekatateam
```

Comment:
"https://github.com/organizations/gekatateam" doesn't seem to be a valid link - should it be "https://github.com/gekatateam" instead? Please let us know and update @M0rdecay - thanks

Reply:
Yes, you're right! Fixed

Reply:
thanks @M0rdecay